phpunit/functional/TicketTest.php

```diff
@@ -7296,4 +7296,133 @@ public function testComputeDefaultValuesForAdd(): void
         $this->assertGreaterThan(0, $id);
         $this->checkInput($ticket, $id, $expected);
     }
+
+    public function isCategoryValidProvider(): array
+    {
+        $ent0 = getItemByTypeName('Entity', '_test_root_entity', true);
+        $ent1 = getItemByTypeName('Entity', '_test_child_1', true);
+
+        // Create Categories
+        $category1 = $this->createItem('ITILCategory', [
+            'name' => 'category_root_entity_recursive',
+            'entities_id' => $ent0,
+            'is_recursive' => 1,
+        ]);
```

Comment:
Creating items in data providers is not a good idea, you should pass the `$category_fields` to the test method and create the category directly inside the method.

Reply:
(this is because items created in data providers are not removed from the database after the tests end since we switched to phpunit)